content/homepage/about-me.md

```diff
@@ -4,21 +4,21 @@ weight: 2
 header_menu: true
 ---
 
-Dr. Sari Mansheim is a US-trained internist with 20 years of practice in medicine.
-In her 15 years of practice in Israel, Dr. Mansheim focuses on treating the whole person,
-also known as [Bio Psycho Social Medicine](https://en.wikipedia.org/wiki/Biopsychosocial_model),
-which states that a medical condition is the result of physical, psychological, and social factors.
-Whether attending to children, adolescents, or adults,
-Dr. Mansheim sees listening to the patient as a significant part of the treatment plan.
+Dr. Sari Mansheim is a US-trained internist. She has been in practice for 20 years; 15 of them in Israel.
+Dr. Mansheim focuses on treating the whole person. Whether seeing children, adolescents, or adults,
+for Dr. Mansheim listening to the patient is a significant part of the treatment plan.
+She practices [Biopsychosocial Medicine](https://en.wikipedia.org/wiki/Biopsychosocial_model),
+which recognizes that a patient's condition is the result of physical, psychological, and social factors.
 
-Dr. Mansheim began her practice of medicine as an internist in an OB/GYN practice in New Jersey,
-giving her extensive experience in most areas of women's health issues.
-Today, she practices in Kupat Holim Meuhedet in Ramat Bet Shemesh Gimmel,
-at a non-profit clinic in Jerusalem area, [Bishvilaych](http://bishvilaych.org/), and privately in Bet Shemesh
-for Israeli and foreign citizens, including seminary students.
-Dr. Mansheim has helped women of all backgrounds through the range of issues that women can face with their health,
-in particular the variety of methods to manage family planning situations.
+Dr. Mansheim practices in Kupat Holim Meuhedet in Ramat Beit Shemesh Gimmel,
+at the non-profit [Bishvilaych](http://bishvilaych.org/) clinic in Jerusalem, 
+and privately in Beit Shemesh for Israeli and foreign citizens, including seminary students. 
+She has extensive experience in most areas of women's health, in particular the 
+variety of methods to manage family planning situations.
 
-Dr. Mansheim is also experienced with alternative treatment methods,
-from herbal to spiritual, all under the guidance of evidence-based medicine.
-Additionally, she has an list of healthcare professionals that she can refer to for treatments.
+Dr. Mansheim practices evidence-based medicine and is also experienced 
+with alternative treatment methods, from herbal to spiritual.
+When appropriate, she refers to associate healthcare professionals for specialized treatments.
+Dr. Mansheim began her career as an internist in an OB/GYN practice in New Jersey. 
```

Comment:
Does it seem like this sentence is out of order because we talked about her work history at the beginning?